deploy/operator/handler.yaml

```diff
@@ -0,0 +1,210 @@
+---
```

Comment:
I think we have to split manifests
deploy/operator/...
deploy/operand/...

For example operator and operand RBAC or namespace can be different.